demos/chatroom/experiment.py

```diff
@@ -1,11 +1,19 @@
 """Coordination chatroom game."""
 
 import dallinger as dlgr
+from dallinger.config import get_config
 try:
     unicode = unicode
 except NameError:  # Python 3
     unicode = str
 
+config = get_config()
+
+
+def extra_settings():
+    config.register('network', unicode)
+    config.register('n', int)
```

Comment:
Definitely, though I'm not entirely convinced this stringency about registering new parameters is a good idea.